x-pack/plugins/siem/public/components/page/network/kpi_network/mock.ts

```diff
@@ -14,22 +14,22 @@ export const mockData: { KpiNetwork: KpiNetworkData } = {
     uniqueSourcePrivateIps: 383,
     uniqueSourcePrivateIpsHistogram: [
       {
-        x: '2019-02-09T16:00:00.000Z',
+        x: 1549728000000,
```

Comment:
Optional: I would have done this so we can see what the original date time value is and know that this x value is a date time:

```ts
new Date('2019-02-09T16:00:00.000Z').valueOf()
```

At the moment it's difficult for this mock to tell that this value is a date time string.

Reply:
Yes, that would be easier to read, thanks! Happy to update this 👍